src/MainWindow.vala

```diff
@@ -188,6 +196,15 @@ namespace PantheonCalculator {
             basic_grid.attach (button_ans, 2, 5, 1, 1);
             basic_grid.attach (button_calc, 3, 5, 1, 1);
 
+            var button_ms = new Button ("MS", _("Store value in memory"));
+            button_mr = new Button ("MR", _("Use value from memory"));
```

Comment:
"Use " is not really appropriate - all this button does is to write the value at the cursor position. "Recall" is maybe better. Maybe nice to include actual value?